apis/python/src/tiledbsoma/_read_iters.py

```diff
@@ -418,28 +465,62 @@ def _cs_reader(
 class SparseTensorReadIterBase(somacore.ReadIter[_RT], metaclass=abc.ABCMeta):
     """Private implementation class"""
 
-    def __init__(self, sr: clib.SOMAArray, shape: NTuple):
-        self.sr = sr
+    def __init__(
+        self,
+        array: SparseNDArray,
+        coords: options.SparseDFCoords,
+        shape: NTuple,
+        result_order: clib.ResultOrder,
+        platform_config: Optional[options.PlatformConfig],
+    ):
+        self.array = array
+        self.coords = coords
         self.shape = shape
+        self.result_order = result_order
+        self.platform_config = platform_config
+
+        sr = array._handle._handle
+
+        if platform_config is not None:
+            ctx = clib.SOMAContext(platform_config)
```

Comment:
If I am understanding this correctly, if I have already opened the array with a context, and I specify a read with a `platform_config`, it creates a new context and ignores the one previously specified in the `open`.

Are these semantics correct?  Vs. (say) modifying the existing context config?

I'm actually not sure what is correct, as I am ignorant on the use case for passing a `platform_config` into a read (I always set up my contexts in the `open` call).

Reply:
I looked at the old code in _dataframe.py, and it was taking  pains to keep the config from the context specified in open:
```
        if platform_config is not None:
            config = context.tiledb_config.copy()
            config.update(platform_config)
            context = clib.SOMAContext(config)
```

So we definitely are changing the behavior, for at least Dataframe

Reply:
Thanks for catching this. I'll take the `sr.context().config()` and then `update` that with the user's passed in config tomrrow.

Reply:
Suggestion - seems like our test coverage missed this, so may want to add at least a quick/simple test to make sure that the correct behavior is codified in a unit test.